packages/kbn-content-management-utils/src/saved_object_content_storage.ts

```diff
@@ -135,6 +135,7 @@ export interface SOContentStorageConstrutorParams<Types extends CMCrudTypes> {
   updateArgsToSoUpdateOptions?: UpdateArgsToSoUpdateOptions<Types>;
   searchArgsToSOFindOptions?: SearchArgsToSOFindOptions<Types>;
   enableMSearch?: boolean;
+  mSearchAdditionalSearchFields?: string[];
```

Comment:
I am still a bit puzzeled why we allow any field name to be passed here instead of having a set of fixed fields to be added to the search (e.g. 'name' | 'description' )

Reply:
`title` and `description` and common, but only `DataView` has `name` and wants it searchable. This is so data view can configure the `name` that belongs to them without hardcoding it